### PR TITLE
chore: simplify issue template

### PR DESCRIPTION
**Pull request by jelbourn**
R: @kara @robertmesserle 

Our current issue template is somewhat noisy, and I think this discourages people from actually following it. I want to try this simpler template and see if it has an affect on people using it.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,35 +1,22 @@
...
+Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
+Plunker template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd
 
-Plunker Template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd?p=preview
 
+#### What is the use-case or motivation for changing an existing behavior?
 
-* **What is the expected behavior?**
 
+#### What are the relevant versions of Angular, Material, OS, affetced browsers, etc.?
```
Comment (kara) [inline .github/ISSUE_TEMPLATE.md]: Typo: affetced --> affected

Comment (jelbourn) [inline .github/ISSUE_TEMPLATE.md]: Fixed.

**Comment by hansl** [comment]
Platform / Browser / Material2 version?

**Comment by jelbourn** [comment]
@hansl it's in there- did you have something specific in mind for how to change it?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,35 +1,22 @@
...
+Providing a Plunker (or similar) is the *best* way to get the team to see your issue.
+Plunker template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd
 
-Plunker Template: http://plnkr.co/edit/o077B6uEiiIgkC0S06dd?p=preview
 
+#### What is the use-case or motivation for changing an existing behavior?
 
-* **What is the expected behavior?**
 
+#### What are the relevant versions of Angular, Material, OS, affected browsers, etc.?
```
Comment (hansl) [inline .github/ISSUE_TEMPLATE.md]: `#### What browser are you using? What's your Material and Angular versions?`

Just that "relevant versions" is up for interpretation and people will just say "Firefox" or omit versions they don't think is relevant.

Comment (jelbourn) [inline .github/ISSUE_TEMPLATE.md]: I think "What browser are you using" is more likely to end up with "Firefox" than "version of affected browsers".

**Pull request merged**